src/components/typography/index.js

```diff
@@ -0,0 +1,3 @@
+import Types from './types';
+
+export default Types;

```

Comment:
Should be at `.ts` file, not `.js`

Also, it does not matter functionally, but `types` is typically reserved for Typescript types, e.g. `export type ...`, so I would not call the file or the export `Types` :).  Maybe, `TypesPage` or something

Reply:
The extension has been changed to ts.